app/views/layouts/_boards_menu_section.html.erb

```diff
@@ -1,5 +1,20 @@
-<% boards.each do |board| %>
-  <li class="nav-item<%= board.id == @board.id ? ' active' : '' unless @board.nil? %>">
-    <%= link_to board.name, board_path(board), class: 'nav-link' %>
+<% if @boards.length > 5 %>
+  <li class="nav-item dropdown">
+    <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      Boards
+    </a>
+    <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+      <% boards.each do |board| %>
+        <li class="<%= board.id == @board.id ? 'active' : '' unless @board.nil? %>">
```

Comment:
WDYT About : 
```
<li class="<%= board.id == @board&.id ? 'active' : '' %>">
```

Safe navigator `&` is meant for this, and prevents having an extra presence check. Also, I'd recommend to prefer "if" as often as possible (in your case, you could do `if @board.present?`)

Reply:
That WFM, I simply copied what was there and removed the extra class so it'd work in the bootstrap dropdown. I'll change both instances.